src/main/scala/com/cloudera/sparkts/TimeSeries.scala

```diff
@@ -137,6 +195,16 @@ class TimeSeries(val index: DateTimeIndex, val data: DenseMatrix[Double],
     }
   }
 
+  def toSamples(): IndexedSeq[(DateTime, Vector[Double])] =
+  {
```

Comment:
Are these new methods related to this change?  If not, mind including them in a separate PR?